packages/core/android/src/main/java/io/sentry/react/RNSentryModuleImpl.java

```diff
@@ -415,12 +416,36 @@ private SentryReplayOptions getReplayOptions(@NotNull ReadableMap rnOptions) {
       androidReplayOptions.addMaskViewClass("com.horcrux.svg.SvgView"); // react-native-svg
     }
 
+    if (rnMobileReplayOptions.hasKey("screenshotStrategy")) {
+      final String screenshotStrategyString = rnMobileReplayOptions.getString("screenshotStrategy");
+      final ScreenshotStrategyType screenshotStrategy =
+          parseScreenshotStrategy(screenshotStrategyString);
+      androidReplayOptions.setScreenshotStrategy(screenshotStrategy);
+    }
+
     androidReplayOptions.setMaskViewContainerClass(RNSentryReplayMask.class.getName());
     androidReplayOptions.setUnmaskViewContainerClass(RNSentryReplayUnmask.class.getName());
 
     return androidReplayOptions;
   }
 
+  private ScreenshotStrategyType parseScreenshotStrategy(@Nullable String strategyString) {
+    if (strategyString == null) {
+      return ScreenshotStrategyType.PIXEL_COPY;
+    }
+
+    try {
+      switch (strategyString.toLowerCase(Locale.ROOT)) {
+        case "canvas":
+          return ScreenshotStrategyType.CANVAS;
+        default:
+          return ScreenshotStrategyType.PIXEL_COPY;
+      }
+    } catch (Exception e) {
+      return ScreenshotStrategyType.PIXEL_COPY;
+    }
```

Comment:
nit: Following up on https://github.com/getsentry/sentry-react-native/pull/5301/files#r2461084612
```suggestion
    switch (strategyString.toLowerCase(Locale.ROOT)) {
      case "canvas":
        return ScreenshotStrategyType.CANVAS;
      default:
        return ScreenshotStrategyType.PIXEL_COPY;
    }
```